### ISSUE TITLE
With Dojo 1.7b2 update, dropping mobile View traps in Chrome

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#601]
@peller @JonFerraiolo

Just updated my local code and tried to drop a **dojox.mobile.View** on to page.  Chrome immediately trapped (showing unhappy face icon).

Tried in Safari and I get this error:

```
Error: unterminated string in code text    :8081/maqetta/user/maqettaUser/ws/workspace/lib/dojo/dojo/dojo.js:14
```

I don't see this issue when using the **src** version.  So maybe beta2 was built incorrectly?

**Comment by jhpedemonte** [comment]
OK, got a decent stack trace this time:

``` javascript
removeComments() at dojo.js:1485
hookProvides() at dojo.js:1523
injectModule.xhrCallback() at dojo.js:1117
getText.req.getText() at dojo.js:949
injectModule() at dojo.js:1160
req.getDojoLoader() at dojo.js:1644
define._1.require() at dojo/_base/loader:15
(anonymous function)() at line 1eval() at (internal script):146
dojo.declare.addJavaScript() at app/dojo/../davinci/ve/Context.js:1870
(anonymous function)() at app/dojo/../davinci/ve/Context.js:285
everyOrSome() at app/dojo/./_base/array.js:17
dojo.mixin.every() at app/dojo/./_base/array.js:157
dojo.declare.loadRequires() at app/dojo/../davinci/ve/Context.js:248
dojo.declare._loadType() at app/dojo/../davinci/ve/tools/CreateTool.js:251
dojo.declare._create() at app/dojo/../davinci/ve/tools/CreateTool.js:204
dojo.declare.create() at app/dojo/../davinci/ve/tools/CreateTool.js:200
dojo.declare.onMouseUp() at app/dojo/../davinci/ve/tools/CreateTool.js:122
dojo.declare.onMouseUp() at app/dojo/../davinci/ve/Context.js:1406
dojo.hitch() at app/dojo/./_base/lang.js:128
```

Unfortunately, Chrome had already trapped, so I can't inspect any of the variables.

**Comment by jhpedemonte** [comment]
This was fixed by Dojo 1.7b3 update.
